website/docs/diff.md

```diff
@@ -4,15 +4,18 @@ sidebar_position: 6
 
 # Known differences
 
+<!-- Each point should have a corresponding test file in https://github.com/FerretDB/dance/tree/main/tests/diff -->
+
 1. FerretDB uses the same protocol error names and codes, but the exact error messages may be different in some cases.
 2. FerretDB does not support NUL (`\0`) characters in strings.
 3. Document keys must not contain `$` sign.
 4. Database and collection names restrictions:
    * name cannot start with the reserved prefix `_ferretdb_`;
    * name must not include non-latin letters, spaces, dots, dollars or dashes;
-   * collection name length must be less or equal than 120 symbols, database name length limit is 63 symbols;
    * name must not start with a number;
-   * database name cannot contain capital letters.
+   * database name cannot contain capital letters;
+   * database name length limit is 63 symbols;
```

Comment:
```suggestion
 * database name length cannot be more than 63 characters.
 * collection name length cannot be more than 120 characters.

```

Reply:
The use of characters is preferable to symbols. Symbols can sometimes mean "@#$%^" and so on, which won't be right.